api/src/mailbox.rs

```diff
@@ -566,6 +579,33 @@ impl Response for StashMeasurementResp {}
 // No command-specific input args
 // No command-specific output args
 
+// CERTIFY_KEY_EXTENDED
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct CertifyKeyExtendedReq {
+    pub hdr: MailboxReqHeader,
+    pub certify_key_req: [u8; CertifyKeyExtendedReq::CERTIFY_KEY_REQ_SIZE],
```

Comment:
nit: If you put this at the end it would make it less invasive if we need to increase the size later.

Reply:
Done